packages/terra-clinical-item-view/src/ItemView.jsx

```diff
@@ -59,6 +59,7 @@ const propTypes = {
   endAccessory: PropTypes.node,
   /**
    * An array of terra-clinical-item-display's to be presented.
```

Comment:
Since we're already updating the prop description, can we fix the plural:

```suggestion
   * An array of terra-clinical-item-displays to be presented.
```

Reply:
Fixed: https://github.com/cerner/terra-clinical/pull/916/commits/545df7819c16c911464ded34b38055cf814b790f#diff-a09eb4a886b729c86a83daac740d9a14a92e66fe733a7a907899150b7fdae956R61